helm/cert-exporter-chart/templates/daemonset.yaml

```diff
@@ -40,23 +38,21 @@ spec:
             cpu: 50m
             memory: 50Mi
         volumeMounts:
-        - mountPath: {{ .Values.Installation.V1.Monitoring.CertExporter.CertPath }}
+        - mountPath: /etc/kubernetes/ssl
           name: certs-volume
-        - mountPath: {{ .Values.Installation.V1.Monitoring.CertExporter.TokenPath }}
+        - mountPath: /etc/tokens
           name: tokens-volume
         - mountPath: /etc/ssl/certs/
           name: ca-certs
       volumes:
       - name: certs-volume
         hostPath:
-          path: {{ .Values.Installation.V1.Monitoring.CertExporter.CertPath }}
+          path: /etc/kubernetes/ssl
       - name: tokens-volume
         hostPath:
-          path: {{ .Values.Installation.V1.Monitoring.CertExporter.TokenPath }}
+          path: /etc/tokens
```

Comment:
Having the vault token exporter here doesn't make much sense for tenants, so we should pull it out into a separate exporter soon. See https://github.com/giantswarm/giantswarm/issues/3996 for tracking.